src/functionaltests/resources/InvalidSubscription.json

```diff
@@ -27,6 +28,7 @@
         "type": "ARTIFACT_1",
         "conditions": [
           {"jmespath": "gav.groupId=='com.mycompany.myproduct'"},
```

Comment:
This also needs to be updated. Remove jmespath expression with gav.

Reply:
done